projects/noloan/app/src/main/java/hasadna/noloan/lawsuit/fragments/LawsuitFormFragment.java

```diff
@@ -51,7 +54,13 @@ public View onCreateView(
     receivedDate = rootView.findViewById(R.id.EditText_receivedSpamDate);
     confirmForm = rootView.findViewById(R.id.Button_formFragment_next);
     scrollView = rootView.findViewById(R.id.ScrollView_formFragment);
-
+    searchCompany = rootView.findViewById(R.id.button_formFragment_searchCompany);
+    searchCompany.setOnClickListener(
```

Comment:
Let's try to stay consistent. You are creating `confirmForm` button then `searchCompany` button, but setting the click listener to `search` first.
Either move the `confirm` listener up or move the `search` listener down.
I like to deal with each button then move to the next. So finding it, setting it's listener and what ever else, then moving to the next and doing the same.

Reply:
Sounds good :+1: